From: Sankar P <psankar@novell.com>
Subject: novfs: NCP Fragments can be upto 64k in size.
References: bnc#625965
Patch-mainline: No

Increase the Maximum fragment size declaration, so as to
avoid wrong boundary checks.

Signed-off-by: Sankar P <psankar@novell.com>

diff --git a/fs/novfs/nwcapi.h b/fs/novfs/nwcapi.h
index 4b6fb99..8cd842a 100644
--- a/fs/novfs/nwcapi.h
+++ b/fs/novfs/nwcapi.h
@@ -301,7 +301,7 @@ N_EXTERN_LIBRARY(NWRCODE)
 #define MIN_NUM_REPLIES		1
 #define MAX_NUM_REQUESTS		4096
 #define MIN_NUM_REQUESTS		1
-#define MAX_FRAG_SIZE			4096
+#define MAX_FRAG_SIZE			65536
 #define MIN_FRAG_SIZE			1
 #define MAX_INFO_LEN			4096
 #define MAX_DOMAIN_LEN			MAX_NETWORK_NAME_LENGTH
-- 
1.7.3.2

